src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  padding: $paddings;
+  box-sizing: border-box;
+  border-radius: $border-radius;
+  border: $border-width solid #f3f3f3;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  width: $width;
+
+  &__img {
+    margin: 0 auto;
+  }
+
+  &__item {
+    display: flex;
+    margin-top: 16px;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin: 0 0 4px;
+    line-height: 18px;
+  }
+
+  &__code {
+    color: $text-color;
+    font-size: 10px;
+    font-weight: 400;
+    margin: 0 0 16px;
+  }
+
+  &__rating {
+    font-size: 10px;
+    font-weight: 400;
+    width: 100%;
+    margin: 0 0 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price {
+    width: 100%;
+    align-items: center;
+    margin: 0 0 16px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__lable {
+    color: $text-color;
+    font-size: 12px;
+    font-weight: 400;
+  }
+
+  &__sum {
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__button {
+    width: 100%;
+    display: flex;
+
+    &-text {
+      text-transform: uppercase;
+      font-size: 14px;
+      font-weight: 700;
```

Comment:
We don't use scss nesting for such cases. Only for elements (&__element-name) or modifiers (&--modifier)
```
&__button {}
&__button-text {}
```